pom.xml

```diff
@@ -224,7 +224,6 @@
                 <configuration>
                     <productVersion>${bamboo.version}</productVersion>
                     <productDataVersion>${bamboo.data.version}</productDataVersion>
-                    <productDataPath>../generated-test-resources.zip</productDataPath>
```

Comment:
I never committed the file cause it has my saucelabs credentials in it. I sent it to aseem by email.

you need to remove that line if you don't have the file